AutoMad/madDictionary.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Nov  6 02:36:40 2024
+
+@author: dragon
+"""
+
+from dataclasses import dataclass
+import numpy as np
+import pandas as pd
+import random
+import wordFormer
+
+@dataclass
+class madDictionary:
```

Comment:
General practice most places I've seen is to name classes starting with a capital letter
```suggestion
class MadDictionary:
```